playbooks/hokusai.md

```diff
@@ -29,17 +29,24 @@ Via `curl`:
 curl --silent https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-latest-$(uname -s)-$(uname -m) -o /usr/local/bin/hokusai && chmod +x /usr/local/bin/hokusai
 ```
 
+### Manual Installation with Python / Pip
+
+See https://github.com/artsy/hokusai#setup
+
 #### Configure Hokusai
 
-*Make sure IAM credentials are set in AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY or ~/.aws/credentials!*
+Prerequisite: install the AWS IAM Authenticator plugin
 
 ```
-hokusai configure --kubectl-version 1.10.7 --s3-bucket artsy-citadel --s3-key k8s/config
+curl -L https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v0.4.0/aws-iam-authenticator_0.4.0_$(uname -s | tr '[:upper:]' '[:lower:]')_amd64 -o /usr/local/bin/aws-iam-authenticator
+chmod a+x /usr/local/bin/aws-iam-authenticator
```

Comment:
Nice!!

FYI: There are packages available for AUR (Arch Linux) and Homebrew:

https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html

https://aur.archlinux.org/packages/aws-iam-authenticator-bin/

Reply:
Fancy 🕺

Reply:
Forgot to push 2705cad1aa0f155e38dd2c43a0bd511807281882 before merging but updated in master...